developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -136,6 +136,54 @@ bundle exec rake -T
 == Installing ChromeDriver
 https://chromedriver.chromium.org/[ChromeDriver] is a separate executable that Selenium WebDriver uses to control Chrome. We use ChromeDriver to run the integration tests for Foreman.
 
+[[plugins]]
+== Installing plugins
+In order to use a plugin, you'll need to install its gem.
+
+From source code:
+[source, ruby]
+....
+cd foreman
+echo "gem '<PLUGIN_NAME>', path: '../PLUGIN_PATH'" >> bundler.d/<PLUGIN_NAME>.local.rb
+....
+
+From github:
+[source, ruby]
+....
+cd foreman
+echo "gem '<PLUGIN_NAME>', git: 'https://github.com/theforeman/<PLUGIN_NAME>.git'" >> bundler.d/<PLUGIN_NAME>.local.rb
+....
+
+Then simply run `bundle install` from foreman to install the plugin and its dependencies.
+In case there are node modules dependencies that don't exist in foreman,
+you will need to install them in the plugin via `npm install`.
+Another option is to re-run `npm install` in foreman,
+which will trigger in the end a postinstall script that will install all node modules of plugins.
+
+after you've installed the dependencies,
+run `bundle exec db:migrate` and `bundle exec db:seed` to update the database.
```

Comment:
I think it's supposed to be `bundle exec rake db:migrate` and `bundle exec rake db:seed`.